Formula/weboob.rb

```diff
@@ -65,6 +65,11 @@ class Weboob < Formula
     sha256 "133b031eaf8fd2c9399b78b8bc5b8fcbe4c31e85295749bb17a87cba8f3c3964"
   end
 
+  resource "prettytable" do
+    url "https://files.pythonhosted.org/packages/e0/a1/36203205f77ccf98f3c6cf17cf068c972e6458d7e58509ca66da949ca347/prettytable-0.7.2.tar.gz"
+    sha256 "2d5460dc9db74a32bcc8f9f67de68b2c4f4d2f01fa3bd518764c69156d9cacd9"
+  done
```

Comment:
This should be `end`.
```suggestion
  end
```